### PR TITLE
[opt](file-cache) support system table file_cache_statistics

### PR DESCRIPTION
**Pull request by morningman**
1. Add new system table: `file_cache_statistics`

	This table is used for viewing metrics related to file cache on BE side

	```
	mysql> select * from information_schema.file_cache_statistics limit 10;
	+-------+---------------+----------------------------+--------------------------------+--------------------+
	| BE_ID | BE_IP         | CACHE_PATH                 | METRIC_NAME                    | METRIC_VALUE       |
	+-------+---------------+----------------------------+--------------------------------+--------------------+
	| 10003 | 172.20.32.136 | /mnt/output/be/file_cache/ | disposable_queue_curr_elements |                  0 |
	| 10003 | 172.20.32.136 | /mnt/output/be/file_cache/ | disposable_queue_curr_size     |                  0 |
	| 10003 | 172.20.32.136 | /mnt/output/be/file_cache/ | disposable_queue_max_elements  |             102400 |
	| 10003 | 172.20.32.136 | /mnt/output/be/file_cache/ | disposable_queue_max_size      |        21474836480 |
	| 10003 | 172.20.32.136 | /mnt/output/be/file_cache/ | hits_ratio                     | 0.8539634687001242 |
	| 10003 | 172.20.32.136 | /mnt/output/be/file_cache/ | hits_ratio_1h                  |                  0 |
	| 10003 | 172.20.32.136 | /mnt/output/be/file_cache/ | hits_ratio_5m                  |                  0 |
	| 10003 | 172.20.32.136 | /mnt/output/be/file_cache/ | index_queue_curr_elements      |                  0 |
	| 10003 | 172.20.32.136 | /mnt/output/be/file_cache/ | index_queue_curr_size          |                  0 |
	| 10003 | 172.20.32.136 | /mnt/output/be/file_cache/ | index_queue_max_elements       |             102400 |
	+-------+---------------+----------------------------+--------------------------------+--------------------+
	```

	It will show metrics of file caches on each BE.

2. Add new metrics `hits_ratio_1h` and `hits_ratio_5m` for file cache

	This 2 metrics will show the hit ratio of file cache in recent 1 hour or 5 minutes.
	So that we can know recent hit ratio instead of global historical hit ratio.

### CODE REVIEW

File: be/src/exec/schema_scanner/schema_file_cache_statistics.cpp [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+namespace doris {
+
+std::vector<SchemaScanner::ColumnDesc> SchemaFileCacheStatisticsScanner::_s_tbls_columns = {
+        //   name,       type,          size
+        {"BE_ID", TYPE_BIGINT, sizeof(int64_t), false},
+        {"BE_IP", TYPE_VARCHAR, sizeof(StringRef), false},
+        {"CACHE_PATH", TYPE_VARCHAR, sizeof(StringRef), false},
+        {"METRIC_NAME", TYPE_VARCHAR, sizeof(StringRef), false},
+        {"METRIC_VALUE", TYPE_DOUBLE, sizeof(double), false}};
+
```
Comment (gavinchou) [inline be/src/exec/schema_scanner/schema_file_cache_statistics.cpp:36]: consider make all the types as string to reduce type complexity and make it for extendable.

Comment (morningman) [reply be/src/exec/schema_scanner/schema_file_cache_statistics.cpp]: done

**Comment by morningman** [comment]
run buildall

**Review by suxiaogang223 — APPROVED** [review]
LGTM

**Review by zy-kkk — APPROVED** [review]

**Pull request merged**